src/content/guide/tools-and-features/remote-diagnostics.md

```diff
@@ -0,0 +1,320 @@
+communicate with the cloud. A variety of factors influence its state,
+such as battery state of charge, signal strength, available memory, and
+application firmware that does not exceed enforced rate limits.
+
+{{#if electron}}
+### SIM Card
+Cellular devices rely on a SIM card to facilitate a connection to the
+cellular network. This SIM must be in an active state, allowing the
+device to try to initiate a data session with the network.
+
+### Cellular Network
+
+In addition to the need for an active SIM, the device still must be in
+range of a cell tower to create a healthy connection to the cellular
+network. Particle works with a global network of cellular carriers to
+allow devices to connect virtually anywhere in the world.
+{{/if}}
+
+### Particle Cloud
+
+The health of the Particle Cloud is critical to devices having the ability to
+successfully connect and communicate. There are a few Particle Cloud
+services in particular that directly impact device health and
+communications:
+
+#### Device Service
+
+The Device Service brokers the connection between an IoT device and
+the Particle Cloud. In addition, the Device Service is responsible for
+shuttling of messages to and from a Particle device.
+
+#### API
+
+The API provides a REST interface to allow remote interactions with
+Particle devices and the cloud. This includes calling a [
+function](/reference/api/#call-a-function),
+checking a [variable](/reference/api/#get-a-variable-value),
+or publishing an [event](/reference/api/#publish-an-event)
+that devices subscribe to.
+
+#### Webhooks
+
+The Webhooks service allows for device data to be sent to other
+apps and services. Webhooks also allow for devices to ingest
+information *from* these Internet services.
+
+## Running Diagnostic Tests
+
+Diagnostics tests can be run for a device using the
+<a href="https://console.particle.io" target="_blank">Particle
+Console</a>. To access Remote Diagnostics, click on a device from your
+device list (on the devices view) to visit the device details page. From
+here, click on the Diagnostics tab. This toggles between the Event Logs
+and Remote Diagnostic tests for the device.
+
+<img src="/assets/images/remote-diagnostics/diagnostics-tab.png"/>
+<p class="caption">Remote Diagnostics are available on the Console's
+device details page.</br>Click on the Diagnostics tab to get started.</p>
+
+You are now presented with the relevant connectivity layers as described
+[above](#connectivity-layers). Click the **Run Tests** button to trigger the
+execution of a variety of health checks:
+
+{{#if electron}}
+<img class="full-width"
+src="/assets/images/remote-diagnostics/default-diagnostics-state-cellular.png"/>
+{{else}}
+<img class="full-width"
+src="/assets/images/remote-diagnostics/default-diagnostics-state-wifi.png"/>
+{{/if}}
+
+Running the tests will kick off diagnostics for each layer of the
+connectivity stack. Tests will be run in parallel, but test results
```

Comment:
"but test results" => "and the test results"

Reply:
Fixed